app/src/main/java/com/taylorgirard/comicconvo/tools/ParseApplication.java

```diff
@@ -25,5 +30,8 @@ public void onCreate() {
                 .build()
         );
 
+        ParseInstallation installation = ParseInstallation.getCurrentInstallation();
+        installation.put("GCMSenderId", "389146853598");
```

Comment:
what is this?

Reply:
The sender id is a part of the Firebase set up. I believe it connects the installation to the Firebase app